baseball.py

```diff
@@ -0,0 +1,47 @@
+
+from pyspark import StorageLevel
+from pyspark.ml.feature import CountVectorizer
+from pyspark.sql import SparkSession
+
+# Setup Spark
+spark = SparkSession.builder.master("local[*]").getOrCreate()
+database = "baseball"
+port = "3306"
+user = "root"
+password = "root"
+query = "select * from batter_counts"
+
+df = spark.read.format("jdbc").options(
```

Comment:
I couldn't get this to run because I set up my spark connection differently and I think I'm missing some dependancies. But i swapped out the code for your spark connection and mine so I could make sure the rest of the code works. So this looks right, but I didn't have a way to run it to be sure.